easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
The CMake-recommend way is normally not to change the library name, and instead build and install in different directories.

But apparently, GROMACS people decided to do so, cf file `cmake/gmxManageSuffixes.cmake` in the source tarball. 

Looking more in this file, we can apparently redefine this. I would say that adding `-DGMX_DEFAULT_SUFFIX=OFF -DGMX_LIBS_SUFFIX="" -DGMX_BINARY_SUFFIX=""` should disable this customized prefixes (not tested).

The problem with this is that scripts/build systems/users/... might expect these prefixes.